docs/install/operator/configuring-serving-cr.md

```diff
@@ -482,8 +442,7 @@ spec:
 
 ### Override replicas, labels and annotations
 
-The following KnativeServing resource overrides the `webhook` deployment to have `3` Replicas, the label `mylabel: foo`, and the annotation `myannotations: bar`,
-while other system deployments have `2` Replicas by using `spec.high-availability`.
+The following KnativeServing resource overrides the `webhook` deployment to have `3` Replicas, the label `mylabel: foo`, and the annotation `myannotations: bar`, while other system deployments have `2` Replicas by using `spec.high-availability`.
```

Comment:
s/Replicas/replicas?

Reply:
It's not on the word list but for stuff like Broker, Trigger etc it uses uppercase, see https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/word-and-phrase-list.md

This content wasn't actually changed in this PR anyway, and I think that sort of thing is out of scope tbh